quality-assurance/Ways-Of-Working-With-Thirdparties.md

```diff
@@ -0,0 +1,85 @@
+# Ways of Working with Third Parties
+
+## Overview
+
+This is intended to provide some guidance as to a more productive interation with suppliers though setting out 
+the terms of engagement.
+
+## The when to get involved
+
+suggested when to be involved is below for guidance :-
+  1. the best possible time would be at the intial inception of the Workpackage or SoR discussions with the business. 
+  1. failing being involved right at the start then it would be advisable to get involved prior to the release of the WP / SoR for tender.
+
+## What are the deliverables for QA
+
+For the QA Deliverable there are 3 types of models that need to be taken into account, which are
+1. UKHO teams
+1. Third party colaberations
+1. Independent third party solutions
+
+### UKHO teams
+These will follow all guidlines and deliver the following TSR documentation, at this point the Test Leads confirm that the test approach in the TSR - 
+detailing the tests deemed neccessary for the project, a sample list of test types is provided but refrence out the TSR for the complete list
+  1. Unit tests
+  2. API tests
+  3. Functional tests
+  4. Non functional tests
+  5. security testing
+  6. usability testing
+
+### Third party colaberations
+Where the third parties are working in colaberation with the UKHO then it is expected that they will be following our UKHO standards. 
+A test lead will be assigned to the WP and they will ensure that the TSR and test coverage has been followed
+
+### Independent third party solutions
+these can be split into two forms of solution 
+  1. Bespoke solution 
+  1. COTS
+
+#### Bespoke solution
+The following should be considered for the delivery
+  1. Documentation in the form of a TSR should be delivered at the outset of the work and agreed with the UKHO test leads to be acheiving our standards as a minimum.
+  2. Where appropriate, the WP will produce a Formal Acceptance Test Document (FAT). This needs to be agreed with UKHO test leads to confirm suitable level of coverage.
+  4. A FAT schedule for test execution will need to be agreed so that UKHO can arrange if needed to wittnessed.
+  5. Where appropriate, a Site Acceptance Test document (SAT) will be delivered. This needs to be agreed with UKHO test leads to confirm suitable level of coverage.
+  6. If the solution is part of a wider programme of delivery there will be a need for UKHO involvement to cover the End-2-End flows through the system.
+
+#### COTS

```

Comment:
This section doesn't include any info on testing.  Perhaps we should be seeking evidence of their QA processes (instead of FAT)?  And then we will normally do SAT which tests correct deployment and configuration.